GrandCentralBoard/Widgets/Bonus/Source/BonusSource.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  Created by Krzysztof Werys on 25/02/16.
+//  Copyright © 2016 Krzysztof Werys. All rights reserved.
+//
+
+import Foundation
+import GrandCentralBoardCore
+
+final class BonusSource : Asynchronous {
+    
+    typealias ResultType = Result<[Person]>
+    typealias UpdateResultType = Result<Updates>
+    typealias ImageResultType = Result<UIImage>
```

Comment:
Probably I started writing this before rebase on GCB version with multiple sources. I could change that but it's not cheap :confused: Can we leave it as it is? One day I'll try to refactor this but now I cannot make it :clock1030: